database/src/pool/sqlite.rs

```diff
@@ -404,6 +404,19 @@ static MIGRATIONS: &[Migration] = &[
         alter table pstat_series_with_target rename to pstat_series;
     "#,
     ),
+    Migration::without_foreign_key_constraints(
```

Comment:
I wonder if we should "delay" the tables on the SQLite side until we finish the Postgres implementation? Just to potentially avoid needless migrations (i.e. Postgres could serve as a "testbed" for the DB). I'm not even sure if we'll ever need this table on the SQLite side, tbh.